EjerciciosT15/src/paquetesiete/LecturaArchivo.java

```diff
@@ -43,9 +46,40 @@ public static double leerRegistros() {
             System.exit(1); 
         } // fin de catch
         
+        
         return suma;        
     } // fin del m�todo leerRegistros
     // cierra el archivo y termina la aplicaci�n
+    
+    public static double leerRegistrosDos(){
+        double suma = 0;
+        double sueldo;
+        int contador = 0;
+        
+        try 
+        {
+            Scanner entrada = new Scanner(new File("data/datos3.csv"));
+
+            while (entrada.hasNext()) {
+                String linea = entrada.nextLine();
+                List<String> lista = Arrays.asList(linea.split("\\|"));
+                ArrayList<String> linea_partes = new ArrayList<>(lista);
+                
+                sueldo = Double.parseDouble(linea_partes.get(2));
+                suma = suma + sueldo;
+                contador = contador + 1;    
+            } 
+            entrada.close();
+        } 
+        catch (Exception e) {
+            System.err.printf("Error, revise: %s\n", e);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.